values/values.postgresql.yaml

```diff
@@ -49,6 +49,10 @@ mysql:
 
 postgresql:
   enabled: true
+  image:
```

Comment:
is it possible to use this with an already existing postgres instance? the same way we are passing connection string properties like username/password/hostname/database name. i was trying to not stand up a separate postgresql just for temporal, but instead use a database inside an existing postgres instance hosted elsewhere

would these charts support this as is?

Reply:
yeah I'm already using it like that 

I'm not using the can `values/values.postgresql.yaml`

You can check my terminal logs

I have postgres chart already installed

Then I passed it in the values (check the values in hel get values)

Reply:
I'm going to update `values/values.postgresql.yaml` to avoid confusion

Reply:
got it. so to clarify, i think the issue i had was, if i did `postgres: enabled: false`, i thought image wasn't required/was optional

it seems in the helm chart it's still required/not optional. i'm guessing that was my mistake, yes?

Reply:
yeah I think the postgresql image bloc is required to know which version to pull in the initContainers (for pg_isready tool)

![image](https://github.com/temporalio/helm-charts/assets/60155224/97e8a8d0-85bb-438c-a62c-cd18567cadd9)